packages/salesforcedx-vscode-core/package.json

```diff
@@ -35,7 +35,7 @@
     "@types/chai": "^4.0.0",
     "@types/glob": "^5.0.30",
     "@types/mocha": "2.2.38",
-    "@types/node": "^6.0.40",
+    "@types/node": "^8.9.4",
```

Comment:
What prompted this change?

Reply:
I'm not sure you can/should be using Node 8 API. VS Code insiders is still using Node 7.9.0. See https://github.com/Microsoft/vscode/issues/11660#issuecomment-246265598

Reply:
I am consistent with what we define in `.nvmrc`. Is this the wrong one for our builds and tests?

Reply:
We don't use any of the new Node 8 APIs even if we are running on .nvmrc.
See the discussion at W-4395117.